main.go

```diff
@@ -184,9 +306,20 @@ func main() {
 	go func(results <-chan *zeroconf.ServiceEntry) {
 		fmt.Printf("[scanner] looking for Shelly devices using mDNS (%ds timeout)...\n", int(scanTimeout.Seconds()))
 		for entry := range results {
+			entry := entry
 			if strings.HasPrefix(entry.Instance, "shelly") {
```

Comment:
Sometimes gen2 devices have a camel case name (mine do)

```suggestion
			if strings.HasPrefix(strings.ToLower(entry.Instance), "shelly") {
```

Reply:
Huh, mine are all lowercase, but I guess it can't hurt. Done.